server/api/boundaries/poamDownload.post.ts

```diff
@@ -2,14 +2,17 @@ import intoStream from "into-stream";
 import { sendStream } from "h3";
 import jwt from "jsonwebtoken";
 import { generatePoam } from "../../utils/excelExport/poamExport";
+import { User, Boundary, BoundaryInterface } from "../../../db/models";
 
 export default defineEventHandler(async (event) => {
   const rawToken = getCookie(event, "tirtoken");
   const config = useRuntimeConfig();
   const decodedToken = jwt.verify(rawToken, config.jwt_key) as { [key: string]: any };
 
   const userId = decodedToken.userId;
+  const user = await User.findByPk(userId);
   const body = await readBody(event);
+  const boundary = (await Boundary.findByPk(body.BoundaryId)) as BoundaryInterface;
 
   const poamWorkBook = await generatePoam(body.BoundaryId, isNaN(userId) ? undefined : userId);
```

Comment:
If we move the validation up, then we can just pass the valid User object through to the generatePoam function and save on a db call.  Alternatively, we could pass through precisely whatever attributes we need.